frontend/pages/demarches-simplifiees-rejoindre-protectenvi.vue

```diff
@@ -17,6 +17,9 @@
               <a
                 class="fr-btn fr-btn--lg"
                 href="https://www.demarches-simplifiees.fr/commencer/rejoindre-protect-envi-pour-mieux-lutter-contre-les-depots-sauvages"
+                target="_blank"
+                rel="noreferrer noopener"
+                aria-label="Ouvrir la démarche simplifiée Protect-Envi dans un nouvel onglet"
```

Comment:
aria-label doit reprendre le texte visible, et nouvel onglet est suffisant pour préciser l'action
```suggestion
                aria-label="Je candidate pour ma collectivité (nouvel onglet)"
```